stable_baselines3/common/off_policy_algorithm.py

```diff
@@ -246,6 +246,8 @@ def load_replay_buffer(
             if truncate_last_traj:
                 self.replay_buffer.truncate_last_trajectory()
 
+        self.replay_buffer.device = self.device
```

Comment:
I'm wondering if a better fix would be to recompute `self.device` when unpickling the replay buffer, no?

Reply:
(I meant using pickle magic method, see what is done in `VecNormalize`)

Reply:
actually forget about that, this fix is simpler